integration-cli/docker_cli_import_test.go

```diff
@@ -41,3 +42,49 @@ func TestImportDisplay(t *testing.T) {
 
 	logDone("import - display is fine, imported image runs")
 }
+
+func TestImportFile(t *testing.T) {
+	runCmd := exec.Command(dockerBinary, "run", "-d", "busybox", "true")
```

Comment:
this could be racey. I would suggest that you make it: `docker run --name bb busybox true` and then use `docker export bb` instead of the container ID.